guidelines/pagination.md

```diff
@@ -0,0 +1,60 @@
+# Pagination
+
+Pagination is a feature introduced into the API to manage requests for large amounts of data.
+It enables us to deliver data in a measured way that does not put a strain on performance for all API users.
+We anticipate rolling out new versions of popular 'Get' operations which will incorporate pagination.
+
+We use a method called _cursor pagination_. This takes the following form:
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/{entity}/getAll`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "Limitation":{
+        "Cursor": "e7f26210-10e7-462e-9da8-ae8300be8ab7",
+        "Count": 10
+    }
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `Limitation` | [Limitation](#limitation) | required | Limitation on the quantity of data returned (using cursor pagination). |
+
+#### Limitation
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Cursor` | string | optional | Unique identifier of the item one newer in time order than the items to be returned. If Cursor is not specified, i.e. null, then the latest or most recent items will be returned. Note that the response message will include the identifier of the oldest item in the response, which can then be used in subsequent calls - see _Limitation example_ below. |
+| `Count` | number | required | Count of items to be returned, minimum 1, maximum 1000. |
+
+> **Limitation example:**
+> A request with Cursor set to null and Count set to 10 will return the latest or most recent 10 items, and the value of Cursor in the response will reference the oldest of those 10 items returned.
+> Let's say the value of Cursor returned is "12345", then if a subsequent request is made with Cursor set to "12345" and Count set to 10, then the next oldest 10 items will be returned.
+> This process can be repeated as required to fetch historical data.
+
+### Response
+
+```javascript
+{
+    "Data": [
+        {
+        	...
+        }
+    ],
+    "Cursor": "7f9325f6-ef44-4911-89a8-ae51010a5aa4"
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Data` | ... | required | ... |

```

Comment:
```suggestion
| `Data` | ... | required | The output of operation. |

```

Reply:
So just write in description `Placeholder for the output of operation.` ?

Reply:
OK, I've used "Placeholder for response data (details will vary)"